resources/views/leaderboards/edit.blade.php

```diff
@@ -0,0 +1,223 @@
+                        @if($leaderboard != null)
+                            <a class="btn btn-danger float-right" href="{{ route("leaderboards::delete", ['id'=>$leaderboard->id]) }}">Delete</a>
+                        @endif
+                    </div>
+
+                </div>
+
+            </form>
+
+        </div>
+
+
+        @if ($leaderboard)
+
+            <div class="col-md-5 mb-3">
+
+                <form method="post"
+                      action="{{ route("leaderboards::entries::add")}}"
+                      enctype="multipart/form-data">
+
+                    {!! csrf_field() !!}
+
+                    <input type="hidden" name="leaderboard_id" value="{{ $leaderboard->id }}">
+
+                    <div class="card md-3">
+
+                        <div class="card-header bg-dark text-white">
+                            {{ ($leaderboard == null ? "Add entries" : "Edit entries for Leaderboard: " . $leaderboard->name)}}
+                        </div>
+
+                        <div class="card-body">
+
+                            @if(count($entries) > 0)
+                                <div class="table-responsive">
+
+                                    <table class="table table-sm table-hover mb-0">
+
+                                        <thead>
+                                            <tr>
+                                                <th></th>
+                                                <th>Name</th>
+                                                <th>{{ $leaderboard->points_name }}</th>
+                                                <th></th>
+                                                <th></th>
+                                            </tr>
+                                        </thead>
+
+                                        <tbody>
+                                            @foreach($entries as $entry)
+                                                <tr>
+                                                    <td>#{{ $loop->index+1 }}</td>
+                                                    <td>{{ $entry->user->name }}</td>
+                                                    <td style="max-width: 80px">
+                                                        <i class="{{ $leaderboard->icon }}"></i>
+                                                        <span data-id="{{ $entry->id }}" class="le_points" >
+                                                            {{ $entry->points}}
+                                                        </span>
+                                                    </td>
+                                                    <td style="min-width: 60px">
+                                                        <a data-id="{{ $entry->id }}" class="fa fas fa-lg fa-caret-up ml-2 le_increase"></a>
+                                                        <a data-id="{{ $entry->id }}" class="fa fas fa-lg fa-caret-down ml-1 le_decrease"></a>
```

Comment:
Am I correct that increasing and decreasing by 1 is the only option to edit points? This could get quite tedious if a lot of people got 10 extra points, maybe also add an input field?